aiohttp/http.py

```diff
@@ -66,5 +63,3 @@
 SERVER_SOFTWARE: str = "Python/{0[0]}.{0[1]} aiohttp/{1}".format(
     sys.version_info, __version__
 )
-
-RESPONSES: Mapping[int, Tuple[str, str]] = http.server.BaseHTTPRequestHandler.responses
```

Comment:
@webknjaz Is there any backwards-compatible concerns with removing this? Shall we keep it in the backport, just in case?

Reply:
Dunno. I don't think it should be considered public API. But we don't explicitly use underscored naming, either. FWIW, we're at mercy of https://www.hyrumslaw.com anyway.
So yeah, we could either leave it out in the backport, or just have a scream test :man_shrugging:.

Reply:
Another way could be defining a `__getattr__()` and emitting a warning.

Reply:
Let's just leave it in the backport. Replacing the code with:
https://github.com/python/cpython/blob/main/Lib/http/server.py#L626-L629

So we can still avoid the import.